pom.xml

```diff
@@ -26,10 +26,10 @@
         <version>1.0.5</version>
     </parent>
 
-    <groupId>org.eclipse.ee4j.xml.registry-api</groupId>
-    <artifactId>xml.registry-api-parent</artifactId>
+    <groupId>jakarta.xml.registries-api</groupId>
```

Comment:
Use "jakarta" but leave the "registry" in the GAV

Reply:
Done